src/librbd/ImageWatcher.cc

```diff
@@ -576,6 +576,14 @@ void ImageWatcher<I>::schedule_request_lock(bool use_timer, int timer_delay) {
     return;
   }
 
+  if (is_blocklisted()) {
+    lderr(m_image_ctx.cct) << __func__ << ": "  << this
+                           << " blocklisted waiting for exclusive lock"
+                           << dendl;
```

Comment:
No need to bother with `__func__` in a unique error message:
```suggestion
    lderr(m_image_ctx.cct) << this << " blocklisted waiting for exclusive lock"
                           << dendl;
```

Reply:
Oops! Will fix this.